qcodes/instrument/parameter.py

```diff
@@ -263,6 +268,7 @@ def __init__(self, name: str,
         # Specify time of last set operation, used when comparing to delay to
         # check if additional waiting time is needed before next set
         self._t_last_set = time.perf_counter()
+        self._validate_on_get = False
```

Comment:
in case we foresee that some logic inside __init__ of the superclass is going to depend on this flag - then yes, we should add it to the constructor. Not sure that i can come up with such a use case now.

If we don't add it to the constructor now, a sweet explanation comment about this is very appreciated :)